projects/demo/server.ts

```diff
@@ -6,75 +6,45 @@ import {join} from 'node:path';
 import {APP_BASE_HREF} from '@angular/common';
 import {provideLocation, provideUserAgent} from '@ng-web-apis/universal';
 import {ngExpressEngine} from '@nguniversal/express-engine';
-import type {Express} from 'express';
 import express from 'express';
 
-import {AppServerModule} from './src/main.server';
+import bootstrap from './src/main.server';
 
-// The Express app is exported so that it can be used by serverless Functions.
-export function app(): Express {
-    const server = express();
-    const distFolder = join(process.cwd(), 'dist/demo/browser');
-    const indexHtml = existsSync(join(distFolder, 'index.original.html'))
-        ? 'index.original.html'
-        : 'index';
-
-    // Our Universal express-engine (found @ https://github.com/angular/universal/tree/master/modules/express-engine)
-    server.engine(
-        'html',
-        ngExpressEngine({
-            bootstrap: AppServerModule,
-        }) as any,
-    );
+declare const __non_webpack_require__: NodeRequire;
```

Comment:
All content from this file is copy-pasted from Taiga repo:
https://github.com/taiga-family/taiga-ui/blob/main/projects/demo/server.ts